src/Symfony/Component/Notifier/Bridge/Infobip/README.md

```diff
@@ -0,0 +1,12 @@
+Infobip Notifier
+===============
+
+Provides Infobip integration for Symfony Notifier.
+
```

Comment:
You should probably document the DSN pattern here (we should do the same for all other transports if you have time to contribute that in another PR).

Reply:
Done!